input/musig2.go

```diff
@@ -173,6 +193,64 @@ func MuSig2CombineKeys(allSignerPubKeys []*btcec.PublicKey,
 	return combinedKey, err
 }
 
+// MuSig2CreateContext creates a new MuSig2 signing context.
+func MuSig2CreateContext(privKey *btcec.PrivateKey,
```

Comment:
Cool, I ended up doing something similar in my taproot branch (to be able to have the mock signers just use these methods on raw private keys), though it made a _new_ package to handle a circular dep that popped up.